README.md

```diff
@@ -17,7 +19,11 @@
 [lic-img]: https://img.shields.io/github/license/mashape/apistatus.svg
 [lic-url]: https://github.com/JuliaReach/RangeEnclosures.jl/blob/master/LICENSE
 
-A [Julia](http://julialang.org) package to bound the range of real-valued functions.
+A [Julia](http://julialang.org) package to bound the range of real-valued functions. The following article showcases the basic functionality, highlighting some of the key design choices:
+
+> Luca Ferranti, Marcelo Forets, and Christian Schilling. *RangeEnclosures.jl: A framework to bound function ranges* [Proceedings of the JuliaCon Conferences](https://doi.org/10.21105/jcon.00122) (2024).
+
+See [below](#-how-to-cite) for how to cite it.
```

Comment:
This link/section does not exist. Should we add it or remove this sentence? We can in any case add a CITATION.bib file.

Reply:
Added citation file and paragraph.